scripts/dataset_processing/tts/audio_processing/preprocess_audio.py

```diff
@@ -133,7 +134,7 @@ def main(cfg):
 
     output_dir.mkdir(exist_ok=True, parents=True)
 
-    entries = read_manifest(input_manifest_path)
+    entries = read_manifest(str(input_manifest_path))
```

Comment:
why `str()` is required here for filepath?  Its better to mandate at argparse level

Reply:
I tried to keep the original type. But agreed to mandate at argparse. Will do.

Reply:
Does this method work for us? The source code implies this version of read_manifest is for use with AIStore, and might not support local files. That might also be why it expects a string and not a file system path.

https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/asr/parts/utils/manifest_utils.py#L374

https://github.com/NVIDIA/NeMo/blob/85afe724db65d641988969a5a9ae2e898ba71381/nemo/utils/data_utils.py#L234

Reply:
> Does this method work for us? The source code implies this version of read_manifest is for use with AIStore, and might not support local files. That might also be why it expects a string and not a file system path.

yes, it works for us. there is a if..else logic: https://github.com/NVIDIA/NeMo/blob/main/nemo/utils/data_utils.py#L217-L219